### PR TITLE
P2 edition

### PR DESCRIPTION
**Pull request by bahram5570**
Tell me your opinion, please.

### CODE REVIEW

**Comment by bahram700** [comment]
Why you deleted all of them?

**Comment by bahram5570** [comment]
They had no more use.

File: f1.txt [hunk 1]
```diff
@@ -1,2 +0,0 @@
-Something....
-And something else!
```
Comment (bahram700) [inline f1.txt:2]: You just needed to remove this line!

Comment (bahram5570) [reply f1.txt:2]: Are you sure?

Comment (bahram5570) [reply f1.txt:2]: I have another opinion.

Comment (bahram700) [reply f1.txt:2]: Yes I'm sure.

File: pull-file1.txt [hunk 2]
```diff
@@ -0,0 +1 @@
+123132453464765
```
Comment (bahram700) [inline pull-file1.txt:1]: Use some texts

Comment (bahram5570) [reply pull-file1.txt:1]: OK

**Pull request merged**